include/GoogleSync/GoogleSync.php

```diff
@@ -972,9 +1127,8 @@ public function setTimezone($timezone)
      *
      * @return bool Success/Failure
      */
-    protected function setLastSync($event_local, $gEventId = null)
+    protected function setLastSync(Meeting $event_local, $gEventId = null)
     {
-
         if (isset($gEventId)) {
             $event_local->gsync_id = $gEventId;
```

Comment:
What if $gEventId is null? Please handle it as it could be `null` also.

Reply:
It's allowed to be null. If it is then the record is saved with a synctime, but not a gsync_id.